gradle.properties

```diff
@@ -1,5 +1,5 @@
-#Tue Aug 08 13:13:22 AEST 2017
-grailsVersion=3.2.11
-grailsWrapperVersion=1.0.0
-gormVersion=6.0.12.RELEASE
-gradleWrapperVersion=3.4.1
+grailsVersion=4.0.13
```

Comment:
Please upgrade to 4.1.1 or to any secured version
More info: https://grails.org/blog/2022-07-18-rce-vulnerability.html